{"name": "test_add_student_data", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: 'E:\\\\Automation\\\\Pytest-API\\\\StudentData.json'", "trace": "def test_add_student_data():\n        url = \"http://thetestingworldapi.com/api/studentsDetails\"\n>       file = open('E:\\Automation\\Pytest-API\\StudentData.json', 'r')\nE       FileNotFoundError: [Errno 2] No such file or directory: 'E:\\\\Automation\\\\Pytest-API\\\\StudentData.json'\n\nTestCases\\test_AddNewStudent.py:7: FileNotFoundError"}, "start": 1652782944166, "stop": 1652782944166, "uuid": "f13c34ba-cb3c-4f6e-a5ba-a82a9f2c59c0", "historyId": "f32e0a06450383f53b10f264680741f4", "testCaseId": "2a7b433136acca31d2c5d07dcbeceb79", "fullName": "TestCases.test_AddNewStudent#test_add_student_data", "labels": [{"name": "parentSuite", "value": "TestCases"}, {"name": "suite", "value": "test_AddNewStudent"}, {"name": "host", "value": "DESKTOP-DNTITBP"}, {"name": "thread", "value": "18392-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_AddNewStudent"}]}